packages/expo-yarn-workspaces/index.js

```diff
@@ -78,6 +79,62 @@ exports.createMetroConfiguration = function createMetroConfiguration(projectPath
   };
 };
 
+
+
+/**
+ * Returns a webpack configuration object that:
+ * 
+ *    * transpiles symlinked workspace packages
+ *    * watches for file changes in symlinked workspace packages
+ */
+
+exports.createWebpackConfigAsync = async function createWebpackConfigAsync(env, argv) {
+  let workspacePackagesToTranspile = [];
+  let workspaceRootPath = findYarnWorkspaceRoot(env.projectRoot);
+
+  if (workspaceRootPath) {
+    debug(`Found Yarn workspace root at %s`, workspaceRootPath);
+
+    const symlinkedModules = getSymlinkedNodeModulesForDirectory(workspaceRootPath);
+    const symlinkedModulePaths = Object.values(symlinkedModules)
+    let workspacePackage = require(path.resolve(workspaceRootPath, 'package.json'));
+
+    // determine if any symlinked modules are a workspace package and include them to be transpiled
+    for (let workspaceGlob of workspacePackage.workspaces) {
+      const workspaceName = workspaceGlob.split('*')[0];
+      const workspacePath = path.resolve(workspaceRootPath, workspaceName);
+
+      for (let actualModulePath of symlinkedModulePaths) {
+        if (isSubDirectory(workspacePath, actualModulePath)) {
```

Comment:
Instead of this logic we should apply the same logic that Yarn does when processing globs, since our intent is to match Yarn 1:1: https://github.com/yarnpkg/yarn/blob/a4708b29ac74df97bac45365cba4f1d62537ceb7/src/config.js#L812-L826. We don't need to implement support for other registries but we should run the glob logic. (Yarn's fs.glob is just the `glob` module.)

Reply:
amazing - thank you I didn't know about this! Will update